runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -415,17 +415,17 @@ private void lookup()
 					label.setText(Integer.toString(combatLevel));
 				}
 			}
-			else if (result.getSkill(skill) != null && result.getSkill(skill).getRank() != -1)
+			else if (result.getSkill(skill) != null)
 			{
 				Skill s = result.getSkill(skill);
 				int level;
 				if (config.virtualLevels() && SKILLS.contains(skill))
 				{
-					level = Experience.getLevelForXp((int) s.getExperience());
+					level = Experience.getLevelForXp((int) Math.max(0, s.getExperience()));
 				}
 				else
 				{
-					level = s.getLevel();
+					level = Math.max(0, s.getLevel());
```

Comment:
I think Adam is referring to this line. Why are we doing `Math.max` on the level as well?

Reply:
The API returns -1 if you have 0 clues/bh kills instead of the expected 0, this tidies that up.